### PR TITLE
✨ add first citypage with animal character

### PR DESCRIPTION
**Pull request by mewiemers**
fix add animal characters in components #5

- import images
- build citypage in storybook
- linking between canvas and citypage

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🦘

**Review by christophvogel — APPROVED** [review]
LGTM 💯

**Review by SilviaBarcellini — APPROVED** [review]
All very nice! 👍

File: pages/citypage.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Citypage from "../components/citypage/Citypage";
+
+function City() {
+  return (
+    <>
+      <Citypage />
+    </>
```
Comment (phlgr) [inline pages/citypage.tsx]: Fragments are not needed here

**Review by phlgr — CHANGES_REQUESTED** [review]
Gj 🚀

**Pull request merged**